ui/src/app/services/sign-in.service.ts

```diff
@@ -48,7 +48,8 @@ export class SignInService {
         gapi.auth2.init({
           client_id: this.clientId,
           hosted_domain: config.gsuiteDomain,
-          scope: 'https://www.googleapis.com/auth/plus.login openid profile'
+          scope: 'https://www.googleapis.com/auth/plus.login openid profile' +
+            ' https://www.googleapis.com/auth/cloud-billing.readonly https://www.googleapis.com/auth/cloud-billing'
```

Comment:
Should be safe to remove readonly scope. The other scope subsumes this. Don't remember why I listed these scopes separately in the ticket:

https://cloud.google.com/billing/v1/how-tos/authorizing